<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recources.qrc">
    <normaloff>:/img/Images/logo huffman.png</normaloff>:/img/Images/logo huffman.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
        background-image: url(:/img/Images/send.png);
        background-color: rgb(37,211,102);
        color:white;
        border-style:outset;
        border-radius:18px;
        font-weight:bold;
		transition:1s;
}

QPushButton:hover{
	background-image: url(:/img/Images/sent.png);
	background-color:#1a8e45;
	cursor:pointer;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Sender</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>0</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The original message</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Receiver</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="message_recived">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>0</width>
        <height>30</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>decoded message</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="message_recived_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>0</width>
        <height>30</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>16</horstretch>
        <verstretch>16</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>encoded message</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="sendBT">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>send</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>296</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>enter message</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string> Message </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Recources.qrc"/>
 </resources>
 <connections/>
</ui>
